spec/index.html

```diff
@@ -1053,17 +1052,11 @@ <h3>RDF Dataset Comparison</h3>
       graph <var>NG1</var> and the RDF dataset <var>D2</var> with default graph
       <var>DG2</var> and any named graph <var>NG2</var>)
       are <dfn data-lt="dataset isomorphism" class="export">dataset-isomorphic</dfn> if and only if
-      there is a bijection <var>M</var> between the nodes, triples and graphs in
-      <var>D1</var> and those in <var>D2</var> such that all of the following properties hold:</p>
+      there is a bijection <var>M</var> from [=blank nodes=] to [=blank nodes=],
+      extended as described in <a href="#graph-isomorphism"></a>, such that:</p>
 
     <ul>
-      <li><var>M</var> maps <a>blank nodes</a> to blank nodes;</li>
-      <li><var>M</var> is the identity map on <a>literals</a> and URIs;</li>
-      <li>For every triple &lt;s p o&gt;, <var>M</var>(&lt;s, p, o&gt;)=
-        &lt;<var>M(s)</var>, <var>M(p)</var>, <var>M(o)</var>&gt;;</li>
-      <li>For every graph <var>G</var>={t1, ..., tn},
-        <var>M(G)</var>={<var>M(t1)</var>, ..., <var>M(tn)</var>};</li>
-      <li><var>DG2</var> = <var>M(DG1)</var>; and</li>
+      <li><var>M(DG1)</var> = <var>DG2</var>; and</li>
       <li>&lt;n, G&gt; is in <var>NG1</var> if and only if
```

Comment:
It might be better to switch away from `&lt; .. &gt;` because it is now used around `s,p,o`.

Reusing () - `(A, B)` - or `[ ]`